tools/profileBuilder/model/latest.go

```diff
@@ -31,11 +31,16 @@ func acceptAll(name string) bool {
 	return true
 }
 
-func ignorePreview(name string) (result bool) {
+var previewSubdir = fmt.Sprintf("%spreview%s", string(os.PathSeparator), string(os.PathSeparator))
+
+func ignorePreview(name string) bool {
```

Comment:
hmm not sure if there is a better name, but ignorePreview doesnt really tell me if it should return true or false if it is preview ( maybe `isPreview` )?

Reply:
Agreed the naming is confusing, will add a comment a clean up the names.